### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by lukaw3d**
cc @lvshaoping007

### CODE REVIEW

File: src/popup/pages/RestoreAccount/index.js [hunk 1]
```diff
@@ -53,7 +53,6 @@ class RestoreAccount extends React.Component {
     sendMsg({
       action: WALLET_NEW_HD_ACCOUNT,
       payload: {
-        pwd: this.state.password,
```
Comment (pro-wh) [inline src/popup/pages/RestoreAccount/index.js:56]: will this affect #11?

**Review by pro-wh — COMMENTED** [review]
mostly looks good. a question about RestoreAccount

File: src/popup/pages/RestoreAccount/index.js (hunk 1 above)
Comment (lukaw3d) [reply src/popup/pages/RestoreAccount/index.js:56]: Nope, this was just `undefined` before

Comment (pro-wh) [reply src/popup/pages/RestoreAccount/index.js:56]: oh ok, thanks for the analysis.

we'll have to see separately about the UI code for the recovery in #11.

**Review by pro-wh — APPROVED** [review]
lg. dunno if we want to get lvshaoping's signoff on this

**Comment by lukaw3d** [comment]
Yes; @lvshaoping007 is it okay to remove all this? Did you plan to use any of this?

**Review by tjanez — APPROVED** [review]

**Comment by lvshaoping007** [comment]
i think these change is ok

**Pull request merged**